geometry/include/pcl/geometry/mesh_indices.h

```diff
@@ -40,7 +40,7 @@
 
 #pragma once
 
-#include <pcl/geometry/boost.h>
+#include <boost/operators.hpp>
```

Comment:
Is this for totally_orderered, unit_steppable and additive?

Reply:
Yes, exactly.